integrations/symfony-bundle.rst

```diff
@@ -167,6 +147,39 @@ Authentication
                 plugins: ['httplug.plugin.authentication.my_wsse']
 
 
+List of services
+````````````````
+
++----------------------------------+---------------------------------------------------------------------+
+| Service id                       | Description                                                         |
++==================================+=====================================================================+
+| httplug.message_factory          | Service* that provides the `Http\Message\MessageFactory`            |
++----------------------------------+---------------------------------------------------------------------+
+| httplug.uri_factory              | Service* that provides the `Http\Message\UriFactory`                |
++----------------------------------+---------------------------------------------------------------------+
+| httplug.stream_factory           | Service* that provides the `Http\Message\StreamFactory`             |
++----------------------------------+---------------------------------------------------------------------+
+| httplug.client.[name]            | | This is your HttpClient that you have configured.                 |
+|                                  | | With the configuration below the name would be `acme_client`.     |
++----------------------------------+---------------------------------------------------------------------+
+| httplug.client                   | This is the first client configured or a client named `default`.    |
```

Comment:
If there is a client named "default", this service is an alias to that client, otherwise it is an alias to the first client configured.